recipe/meta.yaml

```
@@ -23,18 +23,41 @@ requirements:
     - wheel
   run:
     - python
-    - python-dateutil
-    - pytz
-    # https://bitbucket.org/mrabarnett/mrab-regex/issues/314/import-error-no-module-named
-    - regex !=2019.02.19,!=2021.8.27
-    - tzlocal
+    - python-dateutil >=2.7.0
+    - pytz >=2024.2
+    - regex >=2024.9.11
+    - tzlocal >=0.2
+
+  run_constrained:
+    - convertdate >=2.2.1
+    - fasttext >=0.9.1
+    - numpy >=1.19.3,<2
+    - langdetect >=1.0.0
+
+# Skipping tests that require optional extras (calendars, fasttext, langdetect),
+# which are not installed by default in the minimal build.
+{% set ignore_tests = "not (" %}
+{% set ignore_tests = ignore_tests + "test_calendars or " %}
+{% set ignore_tests = ignore_tests + "test_fasttext or " %}
+{% set ignore_tests = ignore_tests + "test_langdetect" %}
+{% if win %}
+{% set ignore_tests = ignore_tests + " or test_date_parser" %}
+{% endif %}
+{% set ignore_tests = ignore_tests + ")" %}
 
 test:
+  source_files:
+    - tests
   requires:
     - pip
+    - pytest
+    - parameterized
```

```suggestion
    - parameterized
    - convertdate >=2.2.1
```
Try to test `test_calendars`

---

@vchychyrko-chychyrko Also, try to run tests with `tzdata  # [py=39 or (osx and x86_64)]` in `test.requires` at least for py39 or for all python variants on osx-64